projects/habitat.md

```diff
@@ -0,0 +1,54 @@
+# Habitat
+
+Includes all code for the Habitat Supervisor, CLI, Studio, exporters, etc.
+
+**Slack Channel** TBD
+
+**Triage Schedule**: TBD
+
+### Project Members
+
+#### Project Owners
+
+Christopher Maier
+  - Github: [christophermaier](https://github.com/christophermaier)
```

Comment:
There should be at least 2 owners since these will be the only users that are admins on the repos

Reply:
Ah, gotcha... change incoming!